query/query.go

```diff
@@ -1824,7 +1824,7 @@ func expandSubgraph(ctx context.Context, sg *SubGraph) ([]*SubGraph, error) {
 		}
 
 		switch child.Params.Expand {
-		// It could be expand(_all_), expand(_forward_), expand(_reverse_) or expand(val(x)).
+		// It could be expand(_all_) or expand(val(x)).
```

Comment:
What's val(x)? I have never any example like this

Reply:
This was a variable that gets predicates values from the old `_predicate_` func. To expand a query with those preds. It was like "expand only the preds found in the `_predicate_` func".

Reply:
Pasted from my reply in Reviewable: "It takes a variable (a list of string values) and uses those values as the names of the predicates to expand. We should discuss whether to keep it but I left it untouched for now."